tajo-core/src/main/java/org/apache/tajo/querymaster/Stage.java

```diff
@@ -1415,8 +1434,9 @@ public StageState transition(Stage stage, StageEvent stageEvent) {
             stage.abort(StageState.KILLED);
             return StageState.KILLED;
           } else {
-            LOG.error("Invalid State " + stage.getSynchronizedState() + " State");
-            stage.abort(StageState.ERROR);
+            TajoInternalError error = new TajoInternalError("Invalid State " + stage.getSynchronizedState() + " State");
```

Comment:
@jihoonson 
this else clause seems to never be executed. I will remove this code block for clarify